addons/auth_signup/models/res_users.py

```diff
@@ -207,9 +212,21 @@ def _action_reset_password(self, signup_type="reset"):
                     mail = self.env['mail.mail'].sudo().create({
                         'subject': self.with_context(lang=user_lang).env._('Password reset'),
                         'email_from': user.company_id.email_formatted or user.email_formatted,
+                        'body': body,
                         'body_html': body,
                         **email_values,
                     })
+                    mail.body_html = self.env['mail.render.mixin']._render_encapsulate(
+                        'mail.mail_notification_layout', body, context_record=self,
+                        add_context={
+                            'email_add_signature': True,
+                            'email_notification_force_header': True,
+                            'email_notification_force_footer': True,
+                            'message': mail.mail_message_id,
```

Comment:
``_render_encapsulate`` has been improved recently, normally some values are not necessary anymore (notably message is done, ...), to check :)